src/components/filters/FiltersInfos.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+
+const FiltersInfos = ({
+  filters,
+  description,
+}) => {
+  const appliedFilters = Object.keys(filters).filter(key => filters[key]?.value)
+
+  const getValue = (key) => {
+    const filterValue = filters[key].value
+
+    switch(description[key].type) {
+      case 'SELECT':
+        const valuesArray = [].concat(filterValue)
+        const labels = valuesArray.map((val) => {
+          const option = description[key].options.find(option => option.value === val)
+          return option.label
+        })
+        return labels.join(', ')
+        break;
+      case 'RANGE':
+        if (filterValue.min || filterValue.max)
+          return `${filterValue.min} to ${filterValue.max} `
```

Comment:
Won't this show eg `undefined to 10` if only `max` is defined? Also your `if` condition checks for falsy values, but `0` is a falsy value that we probably want to show.